far2l/filelist.cpp

```diff
@@ -1086,22 +1086,26 @@ int FileList::ProcessKey(int Key)
 			Redraw();
 			return TRUE;
 		case KEY_CTRLINS:      case KEY_CTRLNUMPAD0:
-
-			if (CmdLength>0)
+		case KEY_CTRLSHIFTINS: case KEY_CTRLSHIFTNUMPAD0: case KEY_CTRLC:     // копировать имена
+		case KEY_CTRLALTINS:   case KEY_CTRLALTNUMPAD0:                       // копировать UNC-имена
+		case KEY_ALTSHIFTINS:  case KEY_ALTSHIFTNUMPAD0:  case KEY_ALTSHIFTC: // копировать полные имена
+		    if (CmdLength > 0)
```

Comment:
why if (CmdLength > 0) moved?

Reply:
To unify the logic: if there is anything in a command line, copy it, and not the selected files. 
OTOH, not all of the shortcuts copy the command line... looks like we need more conditions here.